### PR TITLE
Added etang02's file

### PR DESCRIPTION
**Pull request by ghost**
Added my markdown file. Please comment and merge when ready.

### CODE REVIEW

File: fall-2014/etang02.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Eric Tang
```
Comment (afeld) [inline fall-2014/etang02.md:1]: That's funny, a good friend has the same name. :laughing:

**Comment by missuze** [comment]
sweet comment

**Comment by tsjoberg** [comment]
I am taking the class too!

**Comment by raybblin** [comment]
Hi Eric!

**Comment by djrosenbaum** [comment]
Hi Class!

**Comment by rshatkin** [comment]
@missuze _Bro_ hah

**Comment by jonathanhculver** [comment]
Woo comments!

**Comment by smcarrera** [comment]
Hi Eric!

**Comment by missuze** [comment]
@rshatkin :+1:

**Comment by afeld** [comment]
Looks great! :+1:

**Comment by alexoviedo999** [comment]
yup

**Pull request merged**